game/src/core/components/DrawComponent.java

```diff
@@ -100,7 +99,7 @@ public DrawComponent(final IPath path) throws IOException {
     } catch (NullPointerException np) {
       // We convert the "NullPointerException" to a "FileNotFoundException" because the only
       // reason for a NullPointerException is if the directory does not exist.
-      throw new FileNotFoundException("Path " + path + " not found.");
+      throw new FileNotFoundException("Path " + path.pathString() + " not found.");
```

Comment:
```suggestion
      throw new FileNotFoundException("Path " + path + " not found.");
```

nicht nötig, da `game/src/core/utils/components/path/SimpleIPath.java` (inzwischen) toString() implementiert